runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsPlugin.java

```diff
@@ -73,11 +78,16 @@
 	);
 
 	private static final long PRAYER_DRAIN_INTERVAL_MS = 18200;
-	private static final int CRYPT_REGION_ID = 14231;
+	static final int CRYPT_REGION_ID = 14231;
 
 	private LoopTimer barrowsPrayerDrainTimer;
 	private boolean wasInCrypt = false;
 
+	private int barrowsStartingPotential = 0;
```

Comment:
This should also be initialized on plugin startUp, as a player could be inside the crypt when enabling this plugin/feature. Be sure to invoke it on the client thread